deployment_scripts/appimage/build_appimage.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+python_cmd="python3.7"
+APPDIR="$HOME/monkey-appdir"
+INSTALL_DIR="$APPDIR/usr/src"
+
+GIT=$HOME/git
+
+REPO_MONKEY_HOME=$GIT/monkey
+REPO_MONKEY_SRC=$REPO_MONKEY_HOME/monkey
+
+ISLAND_PATH="$INSTALL_DIR/monkey_island"
+MONGO_PATH="$ISLAND_PATH/bin/mongodb"
+ISLAND_BINARIES_PATH="$ISLAND_PATH/cc/binaries"
+
+is_root() {
+  return $(id -u)
+}
+
+has_sudo() {
+  # 0 true, 1 false
+  return $(sudo -nv > /dev/null 2>&1)
+}
+
+handle_error() {
+  echo "Fix the errors above and rerun the script"
+  exit 1
+}
+
+log_message() {
```

Comment:
why have`()` if it can't contain parameters?

Reply:
It's the Bourne syntax instead of the POSIX syntax, i.e. `my_func()` vs `function my_func`. I don't care much. I can change them all if you want.

Reply:
If you're wondering, can probably defer to shellcheck

Reply:
Shellcheck doesn't complain about it. Mostly it complains about my lack of double quotes around variables in or containing paths.

Reply:
No need to change